quadrat/sass/templates/_header.scss

```diff
@@ -4,12 +4,13 @@
 
 .site-header {
 	overflow: inherit;
-	padding: 10px 0 60px; // TODO: Maybe replace with a responsive custom variable?
+	padding-top: var(--wp--custom--margin--vertical);
 
-	@include break-medium() {
-		padding: var(--wp--custom--post-content--padding--left);
+	// The blockGap is used HORIZONTALLY when the viewport is LARGE (in which case the value defined in theme.json is appropriate) 
+	// It needs to be a different value then when the viewport is SMALL and the gap is used VERTICALLY 
+	@include break-small-only() {
+		gap: 2px !important; // !important because Global Styles, which supplies the gap value, is loaded after theme CSS and wins the specificity battle
```

Comment:
Rather than `!important` can we make the selector more specific?

Reply:
I had a higher specificity hack (double-class) initially but the intent didn't seem as obvious.  There's probably a better way than that though.

Reply:
We could use the .wp-site-blocks class

Reply:
done